src/components/uploader/html/Uploader.html

```diff
@@ -41,99 +41,101 @@
         </form>
 
         <!-- Uploader container -->
-        <form class="flc-uploader fl-uploader fl-progEnhance-enhanced" method="get" enctype="multipart/form-data">
+        <div role="application" class="flc-uploader-container">        
```

Comment:
@waharnum we typically add aria via JS so that if JS isn't enabled or the template is present but the component not initialized, the AT doesn't report it with the wrong semantics and state. The `role="application"` here should be added programmatically. Probably just need to add back the call to add it in the JS file.